index.html

```diff
@@ -82,10 +82,12 @@ <h3 class="dev-intro">I'am a Software Developer</h3>
       </ul>
     </section>
 
+
     <!-- WORK SECTION -->
 
     <section class="main-wrap" id="Work-section">
```

Comment:
- [ ] **[REQUIRED CHANGES but OPTIONAL]**

**NOTE:** As you've done all the changes required by the previous reviewer, I'd approve your PR but you should please kindly implement these changes before you merge to meet the project requirement. 
I could see you didn't notice this while implementing the features of your mobile menu. I understand because we're bound to make mistakes but please, ***fix this issue ASAP!***

- The mobile menu links are working except for the `portfolio`. When clicked, it doesn't close the mobile menu. I was able to see where the issue was coming from. In your `on-enent.js` file, It turned out to be that you were not selecting the right class to effect the behavior you needed. Please, kindly make this update below to make it work as expected:

`const portfolioNav = document.querySelector('#Work-section');` to ` const portfolioNav = document.querySelector('#portfolio');
`

**UPDATE FROM THIS** :point_down: 

```
    **const portfolioNav = document.querySelector('#Work-section');**
    portfolioNav.addEventListener('click', (e) => {
      onClickDisappear();
      e.stopPropagation();
    });
```

**UPDATE TO THIS** :point_down: 

```
 const portfolioNav = document.querySelector('#portfolio');
    portfolioNav.addEventListener('click', (e) => {
      onClickDisappear();
      e.stopPropagation();
    });

```

Reply:
@dunnyk I'm glad I'm able to unblock you, my pleasure! :man_dancing: